_source/_posts/2022-06-06-secure-databases-dotnet.md

```diff
@@ -0,0 +1,650 @@
+---
+layout: blog_post
+title: ""
+author:
+by: advocate|contractor
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness|conversion
+---
+## Securing personal identifying information in an Azure Cosmos DB account
```

Comment:
```suggestion

```
you don't need to include the title in the body, it's pulled from the "front-matter"